.github/workflows/tests-torch.yml

```diff
@@ -21,10 +21,9 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
-        pip install ".[arrow]" \
+        pip install ".[arrow,torch,shell]" \
```

Comment:
Why shell here?

Reply:
To enable shell tests (they run only if these packages are there https://github.com/awslabs/gluonts/blob/dev/test/shell/require-packages.txt)